cpp/src/community/detail/common_methods.cuh

```diff
@@ -29,6 +29,7 @@
 #include <cugraph/detail/utility_wrappers.hpp>
 #include <cugraph/graph_functions.hpp>
 
+#include <algorithm>
```

Comment:
Is this header still necessary?  I suspect you've removed things that used it, as no changes I see require that header.

If we do need the header, I'd suggest putting it after the thrust headers below.